common/app/redux/fetch-challenges-epic.js

```diff
@@ -54,11 +54,14 @@ export function fetchChallengesEpic(
   { getState },
   { services }
 ) {
-  return actions::ofType(types.appMounted)
+  return actions.do(console.log)::ofType(
```

Comment:
you must always bind the log function to the console object

Reply:
That shouldn't be there. I was using it see what was going on